src/mock/task.js

```diff
@@ -0,0 +1,68 @@
+import {getRandomInteger} from "../utils.js";
+import {getRandomColor} from "../utils.js";
+
+const generateDescription = () => {
+  const descriptions = [
+    `Изучить теорию`,
+    `Сделать домашку`,
+    `Пройти интенсив на соточку`
+  ];
+
+  const randomIndex = getRandomInteger(0, descriptions.length - 1);
+
+  return descriptions[randomIndex];
```

Comment:
Тут можно чище, без дополнительной переменной. а с комментом выше, вообще в однострочник получится
```suggestion
  return descriptions[getRandomInteger(0, descriptions.length - 1)];
```